docs/FAQs.md

```diff
@@ -0,0 +1,42 @@
+# GitHub for Jira FAQs
+
+### Permissions
+
+**Q:** The permission scope for code and metadata suggests data is read to synchronize development information. Is code stored on Jira? Should I be concerned that, were the app to be compromised, an actor exfiltrate all our code from GitHub?
+
+**A:** Commits, branches, and merges that occur in a connected GitHub repository will be seen on the dev panel in associated Jira issues. Whenever a commit message includes an issue key, it generates an event that is sent to Jira so the issue specified in the commit message can be updated. Our app needs code access to read commit messages and branch names to correctly link your data to your Jira issues.
```

Comment:
Maybe make it more clear in the answer that the app does not store code. The answer explains what is happening, but it's not really answering the question "is code stored in Jira?"